components/LessonCard.tsx

```diff
@@ -10,6 +10,7 @@ import {
 import NavLink from './NavLink'
 import Image from 'next/image'
 import styles from '../scss/lessonCard.module.scss'
+import { SubmissionStatus } from '../graphql'
 
 const ReviewCount: React.FC<ReviewCountProps> = props => {
   const { loading, data } = useQuery(GET_SUBMISSIONS, {
```

Comment:
Should be changed to codegen generated `useSubmissionsQuery`. Or at least it should be properly typed, currently data returned from this query in untyped. 

It's not related to prisma migration but still.

Reply:
Yeah it's unrelated. Don't wanna touch any more stuff as this PR is already big enough.